src/Components/ImagePost.js

```diff
@@ -3,18 +3,19 @@ import "../CSS/ImagePost.css";
 
 const ImagePost = () => {
   return (
+    
     <>
-      <div className="photo">
-        <div className="photoDetails">
-          <div className="photoImage">
+      <div className="image">
+        <div className="imageDetails">
+          <div className="imageImage">
             <img
               src="https://upload.wikimedia.org/wikipedia/commons/thumb/9/9e/Plus_symbol.svg/500px-Plus_symbol.svg.png"
```

Comment:
you should store links in a variable and use them, instead of adding them in your react component.